sktime/transformations/bootstrap/_tsbootstrap.py

```diff
@@ -0,0 +1,134 @@
+    ----------
+    tsbootstrapper : Bootstrapper from tsbootstrap
+        default = SlidingWindowSplitter(window_length=3, step_length=1)
+        The splitter used for the bootstrap splitting.
+
+    Examples
+    --------
+    >>> from sktime.datasets import load_airline
+    >>> from sktime.transformations.bootstrap import TSBootstrapAdapter
+    >>> from tsbootstrap import MovingBlockBootstrap, MovingBlockBootstrapConfig
+    >>> y = load_airline()
+    >>> bootstrap = TSBootstrapAdapter(MovingBlockBootstrap(MovingBlockBootstrapConfig(10, n_bootstraps=10)))
+    >>> result = bootstrap.fit_transform(y)
+    """
+
+    _tags = {
+        "scitype:transform-input": "Series",
+        "scitype:transform-output": "Panel",
+        "scitype:instancewise": True,  # is this an instance-wise transform?
+        "X_inner_mtype": "pd.DataFrame",  # which mtypes do _fit/_predict support for X?
+        "y_inner_mtype": "None",  # which mtypes do _fit/_predict support for y?
+        "capability:inverse_transform": False,
+        "skip-inverse-transform": True,  # is inverse-transform skipped when called?
+        "univariate-only": False,  # can the transformer handle multivariate X?
+        "handles-missing-data": True,  # can estimator handle missing data?
+        "X-y-must-have-same-index": False,  # can estimator handle different X/y index?
+        "enforce_index_type": None,  # index type that needs to be enforced in X/y
+        "fit_is_empty": True,  # is fit empty and can be skipped? Yes = True
+        "transform-returns-same-time-index": False,
+        "python_dependencies": "tsbootstrap",
+    }
+
+    def __init__(
+        self,
+        tsbootstrapper,
+    ):
+        super().__init__()
+        self.tsbootstrapper = tsbootstrapper
+
+    def _transform(self, X, y=None):
+        """Transform X and return a transformed version.
+
+        private _transform containing core logic, called from transform
+
+        Parameters
+        ----------
+        X : Series or Panel of mtype X_inner_mtype
+            if X_inner_mtype is list, _transform must support all types in it
+            Data to be transformed
+        y : Series or Panel of mtype y_inner_mtype, default=None
+            Additional data, e.g., labels for transformation
+
+        Returns
+        -------
+        transformed version of X
+        """
+        bootstrapped_samples = self.tsbootstrapper.bootstrap(X, test_ratio=0)
+
+        bootstrapped_samples = np.concatenate(list(bootstrapped_samples))
+        multi_index = pd.MultiIndex.from_product(
+            [
+                [
+                    f"synthetic_{i}"
+                    for i in range(self.tsbootstrapper.config.n_bootstraps)
+                ],
+                X.index,
+            ]
+        )
+
+        boostrapped_df = pd.DataFrame(
+            bootstrapped_samples, index=multi_index, columns=X.columns
+        )
+
+        X.index = pd.MultiIndex.from_product([["original"], X.index])
```

Comment:
this is going to break, is it not? In general the bootstrap sampes are shorter

Reply:
Mhm.. I used the behavior of the `STLBootstrap` as template. This bootstrapper is including the original values (called `actual`, I would rename to `actual` in this PR)

```python

from sktime.transformations.bootstrap import STLBootstrapTransformer
from sktime.datasets import load_airline
from sktime.utils.plotting import plot_series  # doctest: +SKIP
y = load_airline()  # doctest: +SKIP
transformer = STLBootstrapTransformer(10)  # doctest: +SKI

y_hat = transformer.fit_transform(y)

```

```
                        Number of airline passengers
series_id   time_index                              
actual      1949-01                       112.000000
            1949-02                       118.000000
            1949-03                       132.000000
            1949-04                       129.000000
            1949-05                       121.000000
...                                              ...
synthetic_9 1960-08                       630.055938
            1960-09                       505.544319
            1960-10                       459.539478
            1960-11                       407.284782
            1960-12                       469.909948
```

Reply:
> Mhm.. I used the behavior of the `STLBootstrap` as template. This bootstrapper is including the original values (called `actual`, I would rename to `actual` in this PR)

Yes, but I think that is a special choice rather than user expected, and it complicates the interface. It is also not sth that `tsbootstrap` does (and imo it's the better design choice) - FYI @astrogilda

Reply:
I will take a look on the other bootstrapped in sktime and check how they are behaving. But in that case, I think it would be good to streamline the behavior using deprecation cycles.

Reply:
you are probably right. Safest perhaps is add a parameter `include_actual` or similar and switch its default.

Reply:
should we open an issue? I am guessing that all bootstrapping transformers are in the `bootstrap` folder.